tools/boards.txt.py

```diff
@@ -1420,6 +1442,7 @@ def all_flash_map ():
 
     f512 = collections.OrderedDict([])
     f1m  = collections.OrderedDict([])
+    f2m0 = collections.OrderedDict([])
```

Comment:
Never set to anything, should be removed.

Reply:
Line 1445 removed, thank you!!!